Libplanet.Net/Swarm.BlockCandidate.cs

```diff
@@ -17,20 +17,20 @@ public partial class Swarm<T>
             var checkInterval = TimeSpan.FromMilliseconds(10);
             while (!cancellationToken.IsCancellationRequested)
             {
-                if (BlockCandidateTable.Any())
+                if (BlockCandidateTable.Count > 0)
                 {
                     BlockHeader tipHeader = BlockChain.Tip.Header;
-                    SortedList<long, Block<T>> blocks =
+                    List<Block<T>> blocks =
                         BlockCandidateTable.GetCurrentRoundCandidate(tipHeader);
-                    if (!(blocks is null) && blocks.Count > 0)
```

Comment:
`object is { }` is compact version of `!(object is null)`. https://devblogs.microsoft.com/dotnet/do-more-with-patterns-in-c-8-0/

Reply:
Aha, I misread. You changed correctly. 😓 Ignore this comment, please.